examples/experimental/dagster-airlift/dagster_airlift/core/sensor.py

```diff
@@ -40,6 +44,15 @@ class AirflowPollingSensorCursor:
     dag_query_offset: Optional[int] = None
 
 
+def check_keys_for_asset_keys(
+    repository_def: RepositoryDefinition, asset_keys: Set[AssetKey]
+) -> Iterable[AssetCheckKey]:
+    for assets_def in repository_def.asset_graph.assets_defs:
```

Comment:
Yea I was just trying to avoid the serial loop through all assets/check specs, but maybe that's spilled milk.

Reply:
If it becomes a problem we could memoize it fairly easily